user_guide.md

```diff
@@ -603,7 +605,7 @@ Did you mean this?
 Run 'hugo --help' for usage.
 ```
 
-Suggestions are automatic based on every subcommand registered and use an implementation of [Levenshtein distance](http://en.wikipedia.org/wiki/Levenshtein_distance). Every registered command that matches a minimum distance of 2 (ignoring case) will be displayed as a suggestion.
+Suggestions are automatic, based on existing subcommands and use an implementation of [Levenshtein distance](http://en.wikipedia.org/wiki/Levenshtein_distance). Every registered command that matches a minimum distance of 2 (ignoring case) will be displayed as a suggestion.
```

Comment:
nit: I'm not an english major and not sure if a comma is appropriate here. It reads best to me if instead of a comma we use an 'and'
`Suggestions are automatic and based on existing commands...`

Reply:
Funny because I started out using an `and`, but then when reading the entire sentence, having two `and`s didn't sound right either: "Suggestions are automatic and based... and use..."

How about: 
 "Suggestions are automatically generated based on existing subcommands and use an implementation of Levenshtein distance."

I'll push an update using that formulation and you can let me know if it sounds better.